google/resource_pubsub_subscription_test.go

```diff
@@ -28,6 +30,17 @@ func TestAccPubsubSubscriptionCreate(t *testing.T) {
 	})
 }
 
+// TODO: Add acceptance test for push delivery.
+//
+// Testing push endpoints is tricky for the following reason:
+// - You need a publicly accessible HTTPS server to handle POST requests in order to receive push messages.
+// - The server must present a valid SSL certificate signed by a certificate authority
+// - The server must be routable by DNS.
+// - You also need to validate that you own the domain (or have equivalent access to the endpoint).
+// - Finally, you must register the endpoint domain with the GCP project.
+//
+// An easy way to test this would be to create an App Engine Hello World app. With AppEngine, SSL certificate, DNS and domain registry is handled for us.
+// App Engine is not yet supported by Terraform but once it is, it will provide an easy path to testing push configs.
```

Comment:
Cloud functions is an alternative here that might be quicker to deploy

Reply:
Good point. Are you still working on adding support for those in Terraform?

Reply:
Not actively; don't wait on me

Reply:
I added a note about Cloud Functions being an option too.